src/app/components/playback/playback_control.rs

```diff
@@ -115,18 +115,43 @@ impl PlaybackControl {
         let model = Rc::new(model);
         let debouncer = Debouncer::new();
         let debouncer_clone = debouncer.clone();
-
         let track_position = &widget.track_position;
+
         widget.seek_bar.connect_change_value(
-            clone!(@weak model, @weak track_position => @default-return signal::Inhibit(false), move |_, _, requested| {
+            clone!(@weak model, @weak track_position => @default-return signal::Inhibit(false), move |_, scroll, mut requested| {
+                let mut debounce = 200;
+                match scroll {
+                    gtk::ScrollType::StepForward => {
+                        let duration = model.current_song_duration().unwrap_or(0.0);
+                        requested = if requested + 5000.0 <= duration {
+                            requested + 5000.0
+                        } else { duration };
+                        debounce = 100;
+                    },
+                    gtk::ScrollType::StepBackward => {
+                        requested = if requested >= 5000.0 {
+                            requested - 5000.0
+                        } else { 0.0 };
+                        debounce = 100;
+                    },
+                    _ => (),
```

Comment:
do you think we should handle other scroll types as well ?

Reply:
I can look at this but most are already handled (jump, page foward, page backwards). The rest I can never find a way to activate them

Reply:
ah interesting! so which event triggers the Step* ones? (sorry if I am adding a lot of comments to your PR and asking a lots fo questions, it is an area of gtk that I don't know much about!)

Reply:
okay so wait I'm confused, what user action results in a StepForward/StepBackward event?

Reply:
(might be interesitng to use https://gtk-rs.org/gtk3-rs/stable/latest/docs/gtk/prelude/trait.RangeExt.html#tymethod.set_increments however )

Reply:
okay nevermind, a larger variety of event trigger the step ones, but the others not so much, they might be related to others implementations of Ranges!

Reply:
but I can confirm that set_increments is sufficient, no need to reimplement the stepping logic i think :)

Reply:
Hmm that might work. I must have missed that. I knew the issue was that it was try to send the default page size which was probably 0 but didnt know how to change